### DISCUSSION TITLE
What U.S. state grows coffee beans?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#61] [Galaxy Trivia] [answered]
**Hint:** 🌈🏝

**Comment by Tynab** [comment, marked as answer]
Coffee is not traditionally grown in the United States, as it requires a tropical or subtropical climate to thrive. However, there is a small but growing coffee industry in the state of Hawaii, which is the only U.S. state where coffee is commercially grown. The coffee produced in Hawaii is known as Kona coffee, and it is highly prized for its unique flavor and aroma. Kona coffee is grown on the slopes of the Mauna Loa volcano, on the Big Island of Hawaii, where the climate and soil are ideal for coffee cultivation. While Hawaii is a relatively small producer of coffee compared to other countries, Kona coffee is considered a gourmet product and is in high demand around the world.
